x/hard/keeper/timelock.go

```diff
@@ -1,23 +1,23 @@
 package keeper
```

Comment:
This file is leftover from when hard used to contain the functionality now in incentive. Looks like the methods aren't used anywhere, so it could be deleted.